CONTRIBUTING.md

```diff
@@ -89,7 +89,7 @@ If you're making substantial changes to an existing topic, adding or changing im
 
 5. If needed, make any necessary updates to your branch. Your pull request will be merged into the master branch after it has been reviewed and your changes are approved.
 
-On a certain cadence, we push all commits from the master branch into the live branch, after which you'll be able to see your contribution live at https://docs.microsoft.com/en-us/dynamics365/.
+On a certain cadence, we push all commits from the master branch into the live branch, after which you'll be able to see your contribution live at [Microsoft Dynamics 365 documentation](https://docs.microsoft.com/dynamics365)/.
```

Comment:
Parenthesis is out of place, should be (https://docs.microsoft.com/dynamics365/).